components/EmptyListView.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import autoBindReact from 'auto-bind/react';
+import _ from 'lodash';
+import { connectStyle } from '@shoutem/theme';
+import { Text, Title, View } from '@shoutem/ui';
+import EmptyList from '../assets/images/emptyList.svg'
+
+class EmptyListView extends PureComponent {
+  static defaultProps = {
+    title: "It's empty in here",
+    message: "We couldn't find anything to show...",
+  }
+
+  constructor(props) {
+    super(props);
+
+    autoBindReact(this);
+  }
+
+  resolveImage() {
+    const { image } = this.props;
+
+    if (image && !_.isFunction(image)) {
+      console.warn(`Image must be an SVG file imported as a React component.`);
+    } else if (image && _.isFunction(image)) {
+      return image;
+    }
+
+    return EmptyList;
+  }
+
+  render() {
+    const { message, title, imageStyle, titleStyle, messageStyle } = this.props
+    const EmptyStateImage = this.resolveImage();
+
+    return (
+      <View styleName="vertical h-center">
+        <EmptyStateImage styleName='image' style={imageStyle} />
+        <Title styleName="title" style={titleStyle}>{title}</Title>
```

Comment:
Just a suggestion. You don't need to implement if you don't find it better. I wouldn't go for stylenames here as it's actually a default style. I'd go for something like this <Title style={[style.title, titleStyle]}>{title}</Title>

For me, styleName would be something like "bold" or "italic" in this example, meaning a variation that stands out from default style. So for our components, you would simply override the base style if you need to, instead of editing a variation that you may need later on

Reply:
I've decided to go with your suggestion. It looks simple and it allows toolkit users to tweak style much easier. Thank you!